### DISCUSSION TITLE
N1 5.77系统 如何升级到最新版？

### DISCUSSION THREAD

**Discussion by Java2Boy** [ophub/amlogic-s9xxx-armbian#1488] [Amlogic]
系统一直使用5.77,想升级到最新版
1：系统里跑了一些服务，如何备份相关项目
2：重新做启动盘刷入吗，系统输入armbian-update 无反应

求指点

**Comment by ophub** [comment]
我不会备份项目。重装系统，重新安装项目可能是最好的方法。
